scripts/gulpfile.factory.js

```diff
@@ -60,12 +63,15 @@ module.exports = function factory({
         )
         // Adds autoprefixing to the compiled sass
         .pipe(
-          postcss([postcssCustomProperties(), autoprefixer(browser_support)])
+          postcss([postcssCustomProperties(), autoprefixer({
+            grid: "autoplace",
```

Comment:
:cowboy_hat_face:

Reply:
Is this indicating approval or that I'm a crazy cowboy for letting autoprefixer go to town on grid? 😉

Reply:
You're a crazy cowboy for changing autoprefixer settings in a PR about `files` :)

Reply:
Honestly the file is just copied over from nav, I didn't even know it was in here.